### ISSUE TITLE
Editor displays only plain text

### ISSUE THREAD

**Issue by loisatwood** [seoscribe/editor#1]
at present, the text editor only displays exactly what the user enters as plain text.

would it not be better to display at least an optional preview mode that renders the markup entered?

**Comment by wnda** [comment]
Yeah, that's certainly the endgame. That and one other feature, which I'll save for another issue — #2

**Comment by wnda** [comment]
It's very easy to let this kind of code get very complex very quickly. There's already plenty going on as the user writes.

**Comment by loisatwood** [comment]
yes I did have a look at it seems quite... intense! but easy to follow in spite of everything. i would have used jquery myself. but you strike me as a performance-purist.

**Comment by wnda** [comment]
Yes, that's pretty much exactly why I tend to avoid jQuery. I started before it came along and by the time it emerged I was quite happy working with vanilla. Of course, it has crept into the odd project of mine over the years, especially where other people were involved, but in this instance I'm calling the shots.

I'm bringing someone else on-board with some more JavaScript experience who might have some input. For now let's just focus on getting ideas on the table. You're more than welcome to clone the editor and play around with it, submit a PR if you've spotted something that could be done better. Otherwise, have a think about the other aspects. Perhaps about the forum and whether we should roll our own or just stick fluxbb on Openshift and skin it for mobile.
